hushline/model.py

```diff
@@ -232,15 +231,6 @@ class AuthenticationLog(Model):
     otp_code: Mapped[Optional[str]] = mapped_column(db.String(6))
     timecode: Mapped[Optional[int]]
 
-    __table_args__ = (
```

Comment:
why remove this? Without this I think alembic will attempt to remove the index. If the names no longer match, can we update?

Reply:
That index isn't actually used for queries, so it can be safely removed without impacting performance.

Reply:
I lied lol. `EXPLAIN` was giving me a full table scan when I didn't have enough representative data in it, but after packing the table it uses an index only scan. I'll back this out.